package.json

```diff
@@ -4,56 +4,59 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve  --project=dev",
+    "clean": "rimraf dist/",
+    "start": "ng serve",
     "build": "ng build",
-    "build:app": "cpr dist/saturn-datepicker src/saturn-datepicker && ng build packages && rimraf src/@my",
-    "build:lib": "rimraf dist && ng-packagr -p saturn-datepicker/package.json && cp saturn-datepicker/src/bundle.css saturn-datepicker/src/_theming.scss dist",
-    "publish": "(cd dist; npm publish)",
+    "build:prod": "ng build --prod",
+    "build:lib": "npm run clean && ng run lib:build:production && npm run copy:theming-files",
+    "copy:theming-files": "cp saturn-datepicker/src/bundle.css saturn-datepicker/src/_theming.scss dist/lib/",
+    "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
+    "publish": "npm publish dist/lib/",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "@angular/animations": "8.2.6",
-    "@angular/cdk": "^8.2.0",
-    "@angular/common": "8.2.6",
-    "@angular/compiler": "8.2.6",
-    "@angular/core": "8.2.6",
-    "@angular/forms": "8.2.6",
-    "@angular/material": "^8.2.0",
-    "@angular/platform-browser": "8.2.6",
-    "@angular/platform-browser-dynamic": "8.2.6",
-    "core-js": "^2.6.9",
-    "rxjs": "^6.5.3",
-    "zone.js": "0.9.1"
+    "@angular/animations": "9.0.4",
+    "@angular/cdk": "9.1.0",
+    "@angular/common": "9.0.4",
+    "@angular/compiler": "9.0.4",
+    "@angular/core": "9.0.4",
+    "@angular/forms": "9.0.4",
+    "@angular/material": "9.1.0",
+    "@angular/platform-browser": "9.0.4",
+    "@angular/platform-browser-dynamic": "9.0.4",
+    "core-js": "3.6.4",
+    "rxjs": "6.5.4",
+    "tslib": "1.11.1",
+    "zone.js": "0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.4",
-    "@angular-devkit/schematics": "^7.3.8",
-    "@angular/cli": "8.3.4",
-    "@angular/compiler-cli": "8.2.6",
-    "@angular/language-service": "8.2.6",
-    "@types/jasmine": "~3.0.0",
-    "@types/node": "^10.14.18",
-    "codelyzer": "^5.1.0",
-    "cpr": "^3.0.0",
-    "jasmine-core": "~3.3.0",
-    "jasmine-spec-reporter": "~4.2.0",
-    "karma": "^3.1.4",
-    "karma-chrome-launcher": "~2.2.0",
-    "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^2.1.0",
-    "karma-jasmine": "~2.0.0",
-    "karma-jasmine-html-reporter": "^1.0.0",
-    "moment": "^2.24.0",
-    "ng-packagr": "^5.5.0",
-    "protractor": "^5.4.2",
-    "rimraf": "^2.7.1",
-    "ts-node": "~7.0.0",
-    "tsickle": "0.37.0",
-    "tslib": "^1.9.0",
-    "tslint": "~5.11.0",
-    "typescript": "3.5.3"
+    "@angular-devkit/build-angular": "0.900.4",
+    "@angular-devkit/build-ng-packagr": "0.900.4",
```

Comment:
Note: The library is now built using the Angular CLI, the workspace project configuration is named `lib` and the default project is set to `dev` (so `ng serve`, `ng build` will run against the development application).


To build the library for a release, just run:
```
ng build --project=lib --configuration=production
# or
ng run lib:build:production
```